packages/rich-text-resolver/README.md

```diff
@@ -0,0 +1,115 @@
+# @kontent-ai/rich-text-resolver
+
+[![npm version](https://img.shields.io/npm/v/@kontent-ai/rich-text-resolver?style=flat-square)](https://www.npmjs.com/package/@kontent-ai/rich-text-resolver)
+[![npm downloads](https://img.shields.io/npm/dt/@kontent-ai/rich-text-resolver?style=flat-square)](https://www.npmjs.com/package/@kontent-ai/rich-text-resolver)
+[![Stack Overflow][stack-shield]](https://stackoverflow.com/tags/kontent-ai)
+[![Discord][discord-shield]](https://discord.gg/SKCxwPtevJ)
+
+> **Note:** This is part of the [@kontent-ai/rich-text-resolver](../../README.md) monorepo.
+> For general information and other packages, see the [main README](../../README.md).
+
+Core package for transforming Kontent.ai rich text into structured formats suitable for resolution and rendering in various environments.
+
+## Installation
+
+```bash
+npm i @kontent-ai/rich-text-resolver
+```
+
+## Features
+
+### Parsing rich text HTML to an array of simplified nodes
+
+The tool provides environment-aware (browser or Node.js) `parseHTML` function to transform HTML into an array of `DomNode` trees. Any valid HTML is parsed, including all attributes. Together with built-in transformation methods, this tool is a suitable option for processing HTML and rich text from external sources, to make it compatible with Kontent.ai rich text format. See dedicated [HTML transformer docs](./docs/index.md) for further information.
+
+### Portable text resolution
+
+[Portable Text](https://github.com/portabletext/portabletext) is a universal standard for rich text representation, with tools available for its transformation and rendering in majority of popular frameworks and languages:
+
+- React: [react-portabletext](https://github.com/portabletext/react-portabletext)
+- HTML: [to-html](https://github.com/portabletext/to-html)
+- Svelte: [svelte-portabletext](https://github.com/portabletext/svelte-portabletext)
+- Vue: [vue-portabletext](https://github.com/portabletext/vue-portabletext)
+- Astro: [astro-portabletext](https://github.com/theisel/astro-portabletext)
+
+> [!TIP]
+> We provide wrapper packages that offer default resolutions for tags that are either unsupported or only partially supported in the original packages (`sub` and `sup` tags, images, tables, and links).
+> - [@kontent-ai/rich-text-resolver-html](../rich-text-resolver-html) – HTML resolution and MAPI transformation
+> - [@kontent-ai/rich-text-resolver-react](../rich-text-resolver-react) – React components and helpers
```

Comment:
Is the original portable text package needed to use our react/vue/html package for portable text? If not, I'd recommend using our package straight away instead of just suggesting it in a tip. The other third-party packages should be in a tip. 😁